public_html/wp-content/mu-plugins/4-helpers-wcpt.php

```diff
@@ -8,6 +8,30 @@
  * Helper functions related to the `wordcamp` post type.
  */
 
+/**
+ * Retrieves the post type string of the wordcamp depending on the current network.
+ *
+ * @return string
+ */
+function get_wordcamp_post_type() {
```

Comment:
I think it might be confusing to refer to Pilot Events as WordCamps, since (at least in my mind) they're a separate type of event. What do you think about a more generic name?

_( the same applies to the docblock comments for both of these functions )_

Reply:
Yeah, I agree. It isn't clear how these events will evolve. 

My intuition is telling me that the WordCamp branding is so strong that we'll likely not rebrand these to be something else outwardly. Instead we'll modify what we consider a WordCamp. I think what we are building now will be considered the default WordCamp and what is now `wordcamp` will be something like regional/flagship/featured/official or whatever the language. I think in the future, and maybe its not feasible, but the current `wordcamp` cpt slug could/should change.

Using that logic, I've left everything as is and didn't try to generalize the naming conventions. I could easily be convinced otherwise.

Reply:
Alright, I pulled out direct references to `central` and `wordcamp` replacing it with WordCamp to denote that WordCamps could have different CPTs in [ec1a82d](https://github.com/WordPress/wordcamp.org/pull/937/commits/ec1a82d0a61236894f5db286d50abc58a2282bb5).

I also added some context to the main doc block. Mind taking another look? Does it make sense?